ddtrace/tracer/textmap_test.go

```diff
@@ -480,41 +489,41 @@ func TestEnvVars(t *testing.T) {
 				t.Setenv(k, v)
 			}
 			var tests = []struct {
-				in         TextMapCarrier
-				traceID128 string
-				out        []uint64 // contains [<trace_id>, <span_id>]
+				in  TextMapCarrier
+				tid traceID
+				sid uint64
 			}{
 				{
 					TextMapCarrier{
 						b3TraceIDHeader: "1",
 						b3SpanIDHeader:  "1",
 					},
-					"",
-					[]uint64{1, 1},
+					traceIDFromLowerUint64(1),
+					1,
 				},
 				{
 					TextMapCarrier{
 						b3TraceIDHeader: "20000000000000001",
 						b3SpanIDHeader:  "1",
 					},
-					"0000000000000002",
-					[]uint64{1, 1},
+					traceIDFromLowerUint64(1),
```

Comment:
I would expect this to need to be `traceIDFromUint64s(2, 1)` actually. I'm surprised this works.